ably/ably.go

```diff
@@ -0,0 +1,24 @@
+package ably
+
+func min(i, j int) int {
+	if i < j {
+		return i
+	}
+	return j
+}
+
+func max(i, j int) int {
+	if i > j {
+		return i
+	}
+	return j
+}
+
+func nonil(err ...error) error {
```

Comment:
@kouno If I was to deploy it right now I'll jump on godoc (listing symbols) + oracle (counting references) + bash (exit code 1 if refcount == 0); haven't used anything that works out of a box.